ida-ws/pom.xml

```diff
@@ -1,273 +1,273 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"

```

Comment:
Let's keep line ending conversions out of this PR.
Right now our use of LF and CRLF is inconsistent across different parts of IDA. We should unify this in a separate PR.